.github/actions/upgrade-indexer/upgrade-indexer.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Gets the absolute path of the script, used to load the common.sh file
+function get_absolute_path() {
+    ABSOLUTE_PATH="$( cd $(dirname ${0}) ; pwd -P )"
+}
+
+get_absolute_path
+. ${ABSOLUTE_PATH}/common.sh
+check_system
+check_version
+
+echo "Installing old version of wazuh indexer..."
```

Comment:
This should be Wazuh Indexer or wazuh-indexer

Reply:
Done